### PR TITLE
Fixes GEARPUMP-234 Update LICENSE and related files for 0.8.2-incubat…

### PR DESCRIPTION
**Pull request by kkasravi**
…ing binary release

### CODE REVIEW

**Comment by manuzhang** [comment]
LGTM, can we wait for https://github.com/apache/incubator-gearpump/pull/108 to go in first ?

File: LICENSE.bin [hunk 1]
```diff
@@ -0,0 +1,349 @@
+                                 Apache License
```
Comment (manuzhang) [inline LICENSE.bin]: why the file extension is `bin` ? Is this auto generated ?

File: dev-tools/build [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```
Comment (manuzhang) [inline dev-tools/build]: Is this a `.sh` file ? Do we need license header here ?

File: LICENSE.bin (hunk 1 above)
Comment (kkasravi) [inline LICENSE.bin]: @manuzhang we were dinged during or 0.8.1 release when one of the rejecting apache voters said we shouldn't include binary licenses unless we're releasing binary artifacts. So we need to have a LICENSE file for source and a LICENSE file for binary. In this case when I build the binary tar.gz I copy the LICENSE.bin to LICENSE.

File: dev-tools/build (hunk 2 above)
Comment (kkasravi) [inline dev-tools/build]: @manuzhang good catch. I'll repush.

**Comment by kkasravi** [comment]
@manuzhang Let's wait for #108 as you noted above.

**Comment by kkasravi** [comment]
@manuzhang looks like #108 is in. I'll add the JIRA to CHANGELOG.md and recreate the artifacts. Do you remember if I used 0.8.1.RC[0-9] for version or 0.8.1? I'll look at the github history...

**Pull request closed without merging**